packages/brian/src/front.js

```diff
@@ -85,8 +85,8 @@ export default (part) => {
     points.s3ArmholeSplit = utils.curveIntersectsY(
       points.shoulder,
       points.mirroredShoulderCp1,
-      points.mirroredFrontArmholePitchCp2,
-      points.mirroredFrontArmholePitch,
+      points.mirroredArmholePitchCp2,
```

Comment:
I think I understand this change but want to confirm. `armholePitch` and `armholePitchCp2` are identical to `frontArmholePitch` and `frontArmholePitchCp2` since they are copied at the top of this file. So this change is about being consistent and shouldn't impact the pattern. Is that right?

Reply:
There is an option to offset the `frontArmholePitch` from `armholePitch` look like it got bolted on later.

Reply:
So sorry but I don't understand. Maybe you could just briefly explain the issue this change addresses?

Reply:
Ah sorry - I was also on the the `back` part in my head not the front But still:

`mirroredArmholePitch*` means actually `backMirroredArmholePitch*` which is correct as we extend the front here into the back part. (and not back into the front as it was)

Reply:
It looked correct when testing, but you got be thinking if I missed something.

Reply:
The `base` part defines both `armholePitch` and `frontArmholePitch` points (and their control points). They are not the same, as the front is cut deeper, depending on the `frontArmholeDeeper` option:

https://github.com/freesewing/freesewing/blob/d0d16af60c32defecaa9c64694dc1e557b650c2e/packages/brian/src/base.js#L128-L132

The `front` part overwrites the `armholePitch` points with the values of the `frontArmholePitch`. That is so that people extending the pattern can just use the same `armholePitch` points without having to worry about back or front parts.

This change does not change anything. Because those points are the same.
However, I do think it's better to be explicit about the intent. This is the front part. We're working with the front armhole pitch point here. So as such, I'm not in favor of this change.

What I mean is, this change would mean that just looking at the line, there's a lot of room for misinterpretation. You have to be aware of the line at the top of the file to know that, actually, it's front pitch point anyway.

So while I think it's good to make sure there's a good reason for using these other names. I think in this case there is.

I would be open to have the `base` part define both `frontArmholePitch` and `backArmholePitch` points, and then have a similar mechanism in the `back` part where it copies the relevant points. I guess that would be the most explicit way to go and would require less cognitive overhead for people reading the code.